react/src/components/SiteTile.js

```diff
@@ -3,12 +3,14 @@ import { Link } from 'react-router';
 
 const SiteTile = (props) => {
   return(
-    <div className="column row">
-      <h3> <Link to={`/sites/${props.id}`}>{props.name}</Link> </h3>
-      <img src="https://idagram.files.wordpress.com/2012/12/skjermbilde-2012-12-09-kl-17-38-00.png" width="400"/>
-      <p> URL: <a href="${props.url}">{props.url}</a> </p>
-      <p> {props.description} </p>
-      <Link to='/'> <button type="button" onClick={props.handleDelete}>Delete This Site</button> </Link>
+    <div className="small-12 large-6 columns site-tile">
+
+        <h3> <Link to={`/sites/${props.id}`}>{props.name}</Link> </h3>
+        <img src="https://idagram.files.wordpress.com/2012/12/skjermbilde-2012-12-09-kl-17-38-00.png" width="400"/>
+        <p> URL: <a href="${props.url}">{props.url}</a> </p>
+        <p> {props.description} </p>
```

Comment:
I would instead add this image to photobucket or Amazon S3, to make sure that you arent relying on an external site dependency